sphinx_gallery/gen_rst.py

```diff
@@ -475,19 +490,24 @@ def generate_dir_rst(src_dir, target_dir, gallery_conf, seen_backrefs):
     return fhindex, computation_times
 
 
-def execute_script(code_block, example_globals, image_path, fig_count,
-                   src_file, gallery_conf):
+def execute_codeblock(code_block, example_globals, image_path,
+                      src_file, gallery_conf):
     """Executes the code block of the example file"""
     time_elapsed = 0
     stdout = ''
 
+    # If example is not suitable to run, skip executing its blocks
+    if not example_globals['__sphx_glr_run_example__']:
+        return stdout, time_elapsed
+
     # We need to execute the code
     print('plotting code blocks in %s' % src_file)
```

Comment:
'plotting code blocks in ' is not very informative ... Maybe just removing this line is the way forward?